crates/modelardb_compression/src/models/macaque_v.rs

```diff
@@ -263,6 +315,19 @@ pub fn grid(
     }
 }
 
+// Extract the unbiased exponent from single precision float.
+fn get_exponent(value: f32) -> i32 {
+    let n_bits: u32 = value.to_bits();
+    let exponent_ = ((n_bits >> 23) & 0xff) as i32;
+    exponent_ - 127
+}
+
+// Left shift unsigned 32-bit integer by a given number of places.
```

Comment:
Maybe use the parameter names in backticks instead of unsigned 32-bit integer and a given number of places